### ISSUE TITLE
React Native Fingerprint Scanner.authenticate got 2 arguments, expected 6

### ISSUE THREAD

**Issue by sam9010** [hieuvp/react-native-fingerprint-scanner#158]
i have an error
this error:
![photo_2020-10-26_15-13-09](https://user-images.githubusercontent.com/41944885/97168574-f012cb80-179d-11eb-98fd-54b7213db7bc.jpg)

**Comment by sahil290791** [comment]
I am also facing this issue, but only in the android device running on 5.1

<img width="609" alt="Screenshot 2020-10-31 at 1 23 30 PM" src="https://user-images.githubusercontent.com/7287280/97774304-89662700-1b7c-11eb-9dcb-19cfcffabb84.png">

**Comment by sam9010** [comment]
i run it in my phone , its version android 5.01 it get error,

i think it is not run in android 5

**Comment by ANIBIT14** [comment]
Yes happened with us also, removed support fro fingerprint from 5.1 devices to make it stop, couldn't find what was wrong in code
